helmfile/values/prometheus-blackbox-exporter-sc.yaml.gotmpl

```
@@ -115,3 +115,12 @@ serviceMonitor:
       scrapeTimeout: 30s
       module: http_401
     {{- end }}
+
+hostAliases:
+{{- range .Values.prometheusBlackboxExporter.hostAliases }}
+  - ip: {{ .ip }}
+    hostNames:
+{{- range .hostNames }}
+      - {{ . }}
+{{- end }}
+{{- end }}
```

```suggestion
{{- with .Values.prometheusBlackboxExporter.hostAliases }}
hostAliases:
  {{- range . }}
  - ip: {{ .ip }}
    hostNames:
      {{- range .hostNames }}
      - {{ . }}
      {{- end }}
  {{- end }}
{{- end }}
```

To not have it end up in an unexpected type when empty, and to make the ends clearer.